core/connection/manager.go

```diff
@@ -89,14 +88,14 @@ func (manager *connectionManager) Connect(consumerID, providerID identity.Identi
 		}
 	}()
 
-	err = manager.startConnection(consumerID, providerID, options)
+	err = manager.start(consumerID, providerID, options)
```

Comment:
Start what? because manager is already running.
`manager.start()` -> `manager.startConnection()` or `manager.createConnection()`

Reply:
Maybe wo dont need 2 manager functions for `start` and `startConnection` and we can merge them?